doc/source/api.rst

```diff
@@ -649,3 +649,17 @@ Random
    random.uniform
    random.permutation
    random.choice
+
+Constants
+=========
+
+.. currentmodule:: larray.core.constants
+.. autosummary::
+   :nosignatures:
+   :toctree: _generated/
+
+   nan
+   inf
+   pi
+   e
+   euler_gamma
```

Comment:
Have you generated the doc to check that sphinx has no error with this part?

Reply:
yes, hence my comment about it being difficult...